src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -995,8 +997,14 @@ def _input_handler():
     )
 
 
-def main():
+total_timeslice_files = 24
```

Comment:
Let's call this `total_forcing_timesteps`... that might be duplication of the information in `nts` and `qts_subdivisions`